composer.json

```diff
@@ -78,5 +78,8 @@
             "phpstan/extension-installer": true,
             "tbachert/spi": false
         }
+    },
+    "conflict": {
+        "drupal/entity_browser": "^2.14"
```

Comment:
btw @AaronGilMartinez I plan to drop this and instead use require-dev to require ~2.13.0, this way it does not restrict websites.
We can drop it once the next version of oe_content is released which has different patch per version of a dependency.